practices/structured-data.md

```diff
@@ -0,0 +1,56 @@
+# Structured data
+
+In the context of the web, structured data refers to information describing and classifying the content of a webpage. For example; on a scientific article page, who the author is, what the article title is, and so on. Structured data contributes towards the vision of the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web#Semantic_Web_solutions) as well as providing SEO benefit. There are 4 widely accepted ways to implement structured data:
+- [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD) (a W3C Recommendation)
+- [RDFa](https://en.wikipedia.org/wiki/RDFa) (a W3C Recommendation)
+- [Microdata](https://en.wikipedia.org/wiki/Microdata_(HTML))
+- [Microformat](https://en.wikipedia.org/wiki/Microformat)
+
+
```

Comment:
Maybe we can add:

You can use any [schema.org](http://schema.org/) schema in JSON-LD. After adding a new JSON-LD script to your website, be sure to validate your code using the [Google’s structured data test tool.](https://search.google.com/structured-data/testing-tool)

Reply:
Thanks. Yep, I'm going to add a bit about vocabulary and our choice of schema.org and nice idea about recommending the tool - will do!